test/Microsoft.AspNetCore.Routing.Tests/Template/TemplateBinderTests.cs

```diff
@@ -118,10 +118,10 @@ public class TemplateBinderTests
             // Arrange
             var encoder = new UrlTestEncoder();
```

Comment:
Are tests using `UrlTestEncoder` just for historical reasons? If they succeed when using the new constructor, do that. Use `[Obsolete]` constructors for at most a test or two per class.